src/QMCWaveFunctions/Jastrow/kSpaceJastrow.cpp

```diff
@@ -602,10 +602,10 @@ kSpaceJastrow::ratio(ParticleSet& P, int iat)
 
 /** evaluate the ratio
 */
-inline void kSpaceJastrow::get_ratios(ParticleSet& P, std::vector<kSpaceJastrow::ValueType>& ratios)
+inline void kSpaceJastrow::evaluateRatiosAlltoOne(ParticleSet& P, std::vector<kSpaceJastrow::ValueType>& ratios)
 {
   RealType J1new(0.0);
-  PosType rnew(P.R[0]);
+  const PosType &rnew(P.activePos);
```

Comment:
Does switch to reference gain anything or is it just stylistic?  Local copy of 3 doubles is probably quite light.

Reply:
This is just stylistic.